gensim/models/coherencemodel.py

```diff
@@ -113,16 +113,18 @@ class CoherenceModel(interfaces.TransformationABC):
     def __init__(self, model=None, topics=None, texts=None, corpus=None, dictionary=None,
                  window_size=None, coherence='c_v', topn=10, processes=-1):
         """
-        Args:
```

Comment:
Why this change? We're trying to use the [Google Code Style](https://google.github.io/styleguide/pyguide.html#Comments) consistently. @menshikh-iv